kratos.gid/apps/CompressibleFluid/write/writeProjectParameters.tcl

```diff
@@ -3,10 +3,10 @@ proc ::CompressibleFluid::write::getParametersDict { } {
     set projectParametersDict [dict create]
 
     # Problem data
-    dict set projectParametersDict problem_data [write::GetDefaultProblemDataDict $CompressibleFluid::app_id]
+    dict set projectParametersDict problem_data [write::GetDefaultProblemDataDict [::CompressibleFluid::GetAttribute id]]

```

Comment:
Well the issue was here. There was an extra bracket `]` that made the JSON converter think this dict was all a string.

Same thing was happening for output_processes